### ISSUE TITLE
Fixture name for relative paths to module?

### ISSUE THREAD

**Issue by yanongena** [cloud-custodian/pytest-terraform#48]
In the documentation, you specify the "name" parameter to be used for the fixture. I thought this could be used for when modules aren't in the same dir as the test. 
So for example: 

```
@terraform("../budget", scope="session", name="something")
def test_oci(something):
    assert "test_instance" in something.outputs["core_instance"]
```

But this doesn't work. I get module not found exception. 

So my question is two fold: 
1) What is the purpose of the name parameter and how can we use it? 
2) How can we reference modules that are not in the same dir as the test file?

**Comment by yanongena** [comment]
I created a fork that should fix this issue: https://github.com/cloud-custodian/pytest-terraform/compare/main...yanongena:pytest-terraform:patch-1

**Comment by marcoceppi** [comment]
If you open up a pull request with your fix I'd be happy to review and add it. Thanks!

**Comment by yanongena** [comment]
Created the PR: https://github.com/cloud-custodian/pytest-terraform/pull/50

**Comment by kapilt** [comment]
merged, thanks this will be included in the 0.6.2 release coming out shortly.
